### ISSUE TITLE
gasAbs with pressure < 10 Pa

### ISSUE THREAD

**Issue by DaveOri** [igmk/pamtra#35]
gasAbs raises an error when pressure is smaller than 10 Pa (rosen_98_gasabs.f90: lines 113-118).
It is probably because of the range of validity of the formula, which is fine, at very low pressures probably the absorption is very low and thus the relative error of the model might be high.
But I wonder if it is too extreme to raise an error and interrupt the computation for the entire column.
I mean: if the pressure is too low what happens? Does the model go crazy or maybe it just calculates very little absorption? The latter case should not really introduce errors in PAMTRA, or?

I am working with IFS files whose top layers are often at very small pressures and it seems a little too much to not have computation at all just because of this limit. I can of course cut my domain, but it is a little annoying considering that I have hybrid coordinates and the cutting level is not so easy to spot.

**Comment by mariomech** [comment]
I totally agree, raising a warning/info would be enough.  It is a leftover from the earlier days of PAMTRA. What I did when running into that problem is really cutting the upper levels. I even included an option to some of the unofficial importers.
